### ISSUE TITLE
[ray] tasks running in docker containers are not stopped on local cluster

### ISSUE THREAD

**Issue by schuhschuh** [ray-project/ray#6898]
### What is your question?

I am wondering if this is expected behavior or a bug. I have a configuration file for a `local` cluster with a number of worker nodes, and am using Docker to run both Ray and my hyperparameter search inside Docker containers on these compute nodes.

Running `ray up` properly starts the Docker containers and Ray processes, but when executing `ray down`, it doesn't seem to have the effect of interrupting the parameter search. When shutting down a cluster, wouldn't it make sense to stop all processes? Maybe even stop the Docker containers (though data only written to locations inside the containers would then be lost, unless `--rm` is not used)? Should maybe the `--rm` option be dropped from `docker run`, and `docker stop` be called upon `ray down`? A user that wishes the containers to be removed immediately upon `ray down` could simply add the `--rm` option to the `docker.run_options` in the cluster configuration file.

*Ray version and other system information (Python version, TensorFlow version, OS):*

- Ray version 0.8.0
- Python 3.7.4

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
